checkers/Board.py

```diff
@@ -206,6 +410,22 @@ def is_free_space(self, row, col):
     def is_same_player(self, row, col, player_turn):
         return self.grid[row][col].player == player_turn
 
+    def make_kings_if_any(self, player_turn):
```

Comment:
Implementation of this function is great, however, do we need to loop the grid for checking the piece is king or not? I think maybe we can check in the _move_piece_ function (after this line `self.grid[selected_row][selected_col].position = (selected_row, selected_col)`). Any thoughts? @aungmcs @Awthura @aungpaing98